### PR TITLE
11 | fix: expose isolateVirtualHostsBySslConfig as a values yaml parameter

### PR DESCRIPTION
**Pull request by gunnar-solo**

### CODE REVIEW

File: docs/content/reference/values.txt [hunk 1]
```diff
@@ -400,6 +400,8 @@
 |gateway.serviceAccount.disableAutomount|bool||disable automounting the service account to the gateway proxy. not mounting the token hardens the proxy container, but may interfere with service mesh integrations|
 |gateway.serviceAccount.kubeResourceOverride.NAME|interface||override fields in the generated resource by specifying the yaml structure to override under the top-level key.|
 |gateway.readGatewaysFromAllNamespaces|bool|false|if true, read Gateway custom resources from all watched namespaces rather than just the namespace of the Gateway controller|
+|gateway.isolateVirtualHostsBySslConfig|bool|false|if true, Added support for the envoy.filters.listener.tls_inspector listener_filter when using the gateway.isolateVirtualHostsBySslConfig=true global setting.|
+|gateway.isolateVirtualHostsBySslConfig|bool|false|if true, Added support for the envoy.filters.listener.tls_inspector listener_filter when using the gateway.isolateVirtualHostsBySslConfig=true global setting.|
```
Comment (ben-taussig-solo) [inline docs/content/reference/values.txt:404]: Why is there a duplicate entry here?

**Review by ben-taussig-solo — COMMENTED** [review]
So the reason this doesn't modify install/test/fixtures/settings/disabled_gateway.yaml (which is modified in the master and 1.12.x PRs) is because the gateway/gloo merge first appears in 1.12.x?

File: docs/content/reference/values.txt (hunk 1 above)
Comment (gunnar-solo) [reply docs/content/reference/values.txt:404]: the _primary_ explanation for that centers around me making mistakes

Comment (ben-taussig-solo) [reply docs/content/reference/values.txt]: looks good now!

**Review by ben-taussig-solo — APPROVED** [review]
Looks good!

**Pull request merged**